### DISCUSSION TITLE
Provide helm charts for deploying conductor in kubernetes

### DISCUSSION THREAD

**Discussion by hankquan** [Netflix/conductor#2633] [Ideas]
With helm charts, users could easily deploy conductor and its dependencies in kubernetes cluster.

**Comment by boney9** [comment]
Hi @howaric - This is an interesting idea. We would love to collaborate with you on this if you are up for it.

**Reply by hankquan** [reply]
I'm trying on it since we have such requirement, and will update when I have progress.

**Comment by art-kor** [comment]
This method looks good, but it's not working. Any reason?

**Comment by ZergRushJoe** [comment]
[Conductor_helm_chart.zip](https://github.com/Netflix/conductor/files/9308417/Conductor_helm_chart.zip)
I have a very basic helm chart for this but it has the problem with not queueing work. @art-kor @boney9  you will need to build the server and ui dockerfiles into images to run this

**Reply by egandro** [reply]
Hello.

Is someone willing to help creating a github repo and add some GHA to maintain this?
